libs/sdk-ui-dashboard/src/presentation/topBar/buttonBar/button/selectors.ts

```diff
@@ -16,40 +15,23 @@ import {
     selectIsNewDashboard,
     selectIsReadOnly,
     selectListedDashboardsMap,
-    selectSupportsAccessControlCapability,
-    selectSupportsHierarchicalWorkspacesCapability,
     selectIsShareButtonHidden,
     selectCanEditDashboardPermission,
+    selectCanEditLockedDashboardPermission,
     selectCanShareDashboardPermission,
+    selectCanShareLockedDashboardPermission,
 } from "../../../../model";
 
-export const selectIsEditModeEnable = createSelector(
-    selectDashboardEditModeDevRollout,
-    selectIsReadOnly,
-    (isDevRollout, isReadOnly) => isDevRollout && !isReadOnly,
-);
-
-/**
- * @internal
- */
-export const selectCanEditLockedDashboard = createSelector(
-    selectDashboardLockStatus,
-    selectSupportsHierarchicalWorkspacesCapability,
-    selectCanManageWorkspace,
-    (isLocked, supportsHierarchicalWorkspaces, canManageWorkspace) =>
-        isLocked && !supportsHierarchicalWorkspaces && canManageWorkspace,
-);
-
 /**
  * @internal
  */
 export const selectCanEnterEditMode = createSelector(
-    selectIsEditModeEnable,
+    selectDashboardEditModeDevRollout,
     selectCanEditDashboardPermission,
+    selectCanEditLockedDashboardPermission,
     selectDashboardLockStatus,
-    selectCanEditLockedDashboard,
-    (isEditModeEnable, canManageAnalyticalDashboard, isLocked, canEditLockedDashboard) =>
-        isEditModeEnable && canManageAnalyticalDashboard && (!isLocked || canEditLockedDashboard),
+    (isEditModeEnabled, canEditDashboardPermission, canEditLockedDashboardPermission, isLocked) =>
+        isEditModeEnabled && canEditDashboardPermission && (!isLocked || canEditLockedDashboardPermission),
```

Comment:
FYI: I've removed the capability check as now both backends support permissions. And `canManageWorkspace` was replaced with canEditLockedDashboardPermission (which is `false` on Tiger and `canManageWorkspace` on Bear)